k8s/services/rabbitmq/helm-3.4.cue

```diff
@@ -0,0 +1,294 @@
+		metrics:        #Metrics
+	}
+
+	msgTopologyOperator: {
+		image: {
+			registry:   "docker.io"
+			repository: "bitnami/rmq-messaging-topology-operator"
+			tag:        "1.10.3-scratch-r1"
+			digest:     ""
+			pullPolicy: "IfNotPresent"
+			pullSecrets: [...v1.#LocalObjectReference]
+		}
+		replicaCount: uint | *1
+		topologySpreadConstraints: []
+		schedulerName:                 string | *""
+		terminationGracePeriodSeconds: string | *""
+		hostNetwork:                   string | *"false"
+		dnsPolicy:                     string | *"ClusterFirst"
+
+		livenessProbe: v1.#Probe | *{
+			enabled:             true
+			initialDelaySeconds: 5
+			periodSeconds:       30
+			timeoutSeconds:      5
+			successThreshold:    1
+			failureThreshold:    5
+		}
+		readinessProbe: v1.#Probe | *{
+			enabled:             true
+			initialDelaySeconds: 5
+			periodSeconds:       30
+			timeoutSeconds:      5
+			successThreshold:    1
+			failureThreshold:    5
+		}
+		startupProbe: v1.#Probe | *{
+			enabled:             false
+			initialDelaySeconds: 5
+			periodSeconds:       30
+			timeoutSeconds:      5
+			successThreshold:    1
+			failureThreshold:    5
+		}
+		customLivenessProbe:         v1.#Probe | *{}
+		customReadinessProbe:        v1.#Probe | *{}
+		customStartupProbe:          v1.#Probe | *{}
+		existingWebhookCertSecret:   string | *""
+		existingWebhookCertCABundle: string | *""
+		resources:                   v1.#ResourceRequirements | *{
+			limits: {}
+			requests: {}
+		}
+		podSecurityContext: v1.#PodSecurityContext | *{
+			enabled: true
+			fsGroup: 1001
+		}
+		containerSecurityContext: v1.#PodSecurityContext | *{
+			enabled:                true
+			runAsUser:              1001
+			runAsNonRoot:           true
+			readOnlyRootFilesystem: true
+		}
+		command:          v1.#Command | *[]
+		args:             v1.#Args | *[]
+		fullnameOverride: string | *""
+		hostAliases: []
+		podLabels:             k8s.#Labels
+		podAnnotations:        k8s.#Annotations
+		podAffinityPreset:     string | *""
+		podAntiAffinityPreset: string | *"soft"
+		nodeAffinityPreset:    v1.#Affinity
+		affinity:              v1.#Affinity
+		nodeSelector:          v1.#NodeSelector
+		tolerations: [...v1.#Toleration]
+		updateStrategy: {
+			type: string | *"RollingUpdate"
+		}
+		priorityClassName: string | *""
+		lifecycleHooks: {}
+		containerPorts: {
+			metrics: k8s.#Port | *8080
+		}
+		extraEnvVars: []
+		extraEnvVarsCM:     string | *""
+		extraEnvVarsSecret: string | *""
+		extraVolumes: []
+		extraVolumeMounts: []
+		sidecars: []
+		initContainers: []
+		rbac: {
+			create: bool | *true
+		}
+		serviceAccount: #ServiceAccount
+		metrics:        #Metrics
+
+		service: k8s.Service | *{
+			type:  string | *"ClusterIP"
+			ports: k8s.#ServicePort | *{
+				webhook: 443
+			}
+			nodePorts: k8s.#ServicePort | *{
+				http: ""
+			}
+			clusterIP:      string | *""
+			loadBalancerIP: string | *""
+			extraPorts: []
+			loadBalancerSourceRanges: []
```

Comment:
what's the schema? maybe [...string]

if it's not know we can leave it as anything [..._]